spring-cloud-azure-eventhub-stream-binder/src/main/java/com/microsoft/azure/eventhub/stream/binder/config/EventHubBinderConfiguration.java

```diff
@@ -31,8 +31,6 @@
  */
 @Configuration
 @ConditionalOnMissingBean(Binder.class)
-@AutoConfigureBefore(TelemetryAutoConfiguration.class)
-@AutoConfigureAfter({AzureContextAutoConfiguration.class, AzureEventHubAutoConfiguration.class})
 @EnableConfigurationProperties({AzureEventHubProperties.class, EventHubExtendedBindingProperties.class})
```

Comment:
I think we should remove the usage of `@AutoConfigureBefore` and `@AutoConfigureAfter` from the whole code base. We don't want to get the same feedback for other modules.
I am fine with that we don't fix them in this PR since this PR is all about Event Hub binder. Created #350 to track this work.

Reply:
Agree. I will confirm all things work after removing these in another PR